File: 프로그래머스/0/120868. 삼각형의 완성조건 （2）/삼각형의 완성조건 （2）.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>
int compare(const int*, const int*);

// sides_len은 배열 sides의 길이입니다.
int solution(int sides[], size_t sides_len) {
    int answer = 0;
    qsort(sides, sides_len, sizeof(int), compare);
    answer = (sides[0]+sides[1])-(sides[0]-sides[1])-1;
    return answer;
}
int compare(const int* a, const int* b) {
    return(*b - *a);
}

File: 프로그래머스/0/120868. 삼각형의 완성조건 （2）/삼각형의 완성조건 （2）.java
class Solution {
    public int solution(int[] sides) {
        int c=0;
        
        if(sides[0] < sides[1]) {
            int temp = sides[0];
            sides[0] = sides[1];
            sides[1] = temp;
        }
        for(int i = sides[0]-sides[1]+1; i < sides[0]+sides[1]; i++) {
            c++;
        }
        
        return c;
    }
}